cli/src/command/generate_config.rs

```
@@ -131,6 +131,8 @@ impl ExonumCommand for GenerateConfig {
         let master_key_path = self
             .master_key_path
             .unwrap_or_default()
+            .canonicalize()
+            .unwrap_or_default()
```

Shouldn't we return an error in case `canonicalize` fails?